cnf-certification-test/identifiers/exceptions.go

```diff
@@ -34,6 +34,9 @@ const (
 
 	SecConNonRootUserExceptionProcess = `If your application needs root user access, please document why your application cannot be ran as
 											non-root and supply the reasoning for exception.`
+	SecConExceptionProcess = `If the contaier had the right configuration of the allowed category from the 4 list so the test will pass the 
+	list is on page 51 on the CNF Security Context Constraints (SCC) section 4.5, 
+	Applications MUST use one of the approved Security Context Constraints profiles unless an exception (SEAT) is approved by Verizon.`
```

Comment:
Do we want to explicitly use `Verizon` here?  @mwlinca ?

Reply:
i removed that